scripts/codemod-property-tsdoc.mjs

```
@@ -0,0 +1,369 @@
+#!/usr/bin/env node
+/**
+ * ONE-TIME migration: move prop documentation from the `@property` block above
+ * each `<X>Props` interface onto the interface members as inline TSDoc.
+ *
+ *   /**                                    /**
+ *    * Props for the Footer component.      * Props for the Footer component.
+ *    *                                      *​/
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)